src/v2/view-builder/views/PollView.js

```diff
@@ -1,53 +1,57 @@
 import { loc, View } from 'okta';
 import hbs from 'handlebars-inline-precompile';
-import { BaseForm, BaseView } from '../internals';
+import { BaseFormWithPolling, BaseView } from '../internals';
 import polling from './shared/polling';
 import { MS_PER_SEC } from '../utils/Constants';
 
 const PollMessageView = View.extend({
   template: hbs`
     <div class="ion-messages-container">
-      <p>{{i18n code="poll.form.message" 
+      <p>{{i18n code="poll.form.message"
         bundle="login" arguments="countDownCounterValue" $1="<span class='strong'>$1</span>"}} </p>
     </div>
     <div class="okta-waiting-spinner"></div>
     `
   ,
   getTemplateData() {
-    const countDownCounterValue = this.options;
     return {
-      countDownCounterValue,
+      countDownCounterValue: this.options,
```

Comment:
this is unconventional. `this.options` usually is object hence shall be `const {countDownCounterValue} = this.options;`